packages/docs/celo-gold-holder-guide/voting-validators.md

```diff
@@ -0,0 +1,81 @@
+
+[Validator elections](../celo-codebase/protocol/proof-of-stake/validator-elections.md) are held every epoch (approximately once per day). All 100 slots are available at each election. Validators are selected [in proportion](../celo-codebase/protocol/proof-of-stake/validator-elections.md#Running-the-Election) to votes received for each Validator Group.
+
+If you hold Celo Gold (or are a beneficiary of a [`ReleaseGold` contract](release-gold.md) that allows voting), you can vote for Validator Groups. A single account can split their LockedGold balance to have outstanding votes for up to 10 groups.
+
+Celo Gold that you lock and use to vote for a group that elects one or more validators receives [epoch rewards](../celo-codebase/protocol/proof-of-stake/epoch-rewards.md) every epoch (approximately every day) once the community passes a governance proposal enabling rewards. The initial level of rewards is anticipated to be around 6% per annum equivalent (but is subject to change).
+
+Unlike a number of Proof of Stake protocols, Celo Gold used for voting is never at risk. The actions of the validator groups or validators you vote for can cause you to receive lower or higher rewards, but the Celo Gold you locked will always be available to be unlocked in the future.
+
+## Validator and Validator Group Explorers
+
+The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.
+
+{% hint style="warning" %}
+**Warning**: Exercise caution in relying on validator-supplied names to determine their real-world identity.Malicious participants may attempt to impersonate other validators in order to attract votes.
+
+Validators and groups can also supply [verifiable DNS claims](../operations-manual/validator-explorer.md), and the Celo Validator Explorer displays these. You can use these to securely identify that the same entity has access both to the account of a validator or group and the supplied DNS records.
+{% endhint %}
+
+### [Celo Validator Explorer](https://validators.celo.org) (cLabs)
+
+The Celo Validator Explorer has tabs to show either Release Candidate or Baklava networks.
+
+The list shows Validator Groups and, when you expand each group, the Validators that are affiliated to that group.
+
+A white check mark next to the name of a Validator Group shows that there is one or more DNS metadata claims verified for that group (see below).
+
+The Votes Available column shows:
+
+- On the left: Votes made for the group, as a percentage of the total Locked Gold
+
+- On the right: The voting cap of that group, as a percentage of the total Locked Gold
+
+- In the middle: votes made for the group as a proportion of the voting cap
```

Comment:
It'd be worth explaining the voting cap in the "proof-of-stake election essentials" section

Reply:
I had in a previous draft but moved it below to keep the explanation brief...